storm/src/test/storm/mesos/MesosCommonTest.java

```diff
@@ -317,4 +328,32 @@ public void testExecutorMem() throws Exception {
     expectedResult = 100;
     assertEquals(result, expectedResult, DELTA);
   }
+
+  @Test
+  public void getConsolidatedOfferResourcesPerNode() {
+    RotatingMap<Protos.OfferID, Protos.Offer> r = new RotatingMap(2);
+    Protos.Offer offer = TestUtils.buildOffer("0-1", "h1", 0, 0);
+    r.put(offer.getId(), offer);
+    offer = TestUtils.buildOffer("0-2", "h1", 10, 1000);
+    r.put(offer.getId(), offer);
+    offer = TestUtils.buildOfferWithReservation("0-1", "h1", 0, 0, 200, 2000);
+    r.put(offer.getId(), offer);
+
+    offer = TestUtils.buildOffer("0-3", "h2", 0, 0);
+    r.put(offer.getId(), offer);
+    offer = TestUtils.buildOfferWithPorts("O-4", "h2", 0, 0, 1, 100);
+    r.put(offer.getId(), offer);
+
```

Comment:
`assertEquals` gives us the opportunity to use an epsilon to check equality for doubles (as are used in the representation of CPU and MEM resources), so we should switch to that.